testing/helm/tests/manypvc.yaml

```diff
@@ -0,0 +1,43 @@
+storage:
+  planes:
+  - name: dali
+    pvc: dali-localmany-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/dalistorage"
+    category: dali
+  - name: dll
+    pvc: dll-localmany-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/queries"
+    category: dll
+  - name: sasha
+    pvc: sasha-localmany-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/sasha"
+    category: sasha
+  - name: debug
+    pvc: debug-localmany-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/debug"
+    category: debug
+  - name: manydata
+    pvc: manydata-localmany-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/manydata"
+    category: data
+    numDevices: 5
+    aliases:
+    - pvc: cachedata-localmany-hpcc-localfile-pvc
+      prefix: "/var/lib/HPCCSystems/manydatacache"
+      mode: [ random ]
+      numMounts: 1
+    - pvc: cachedata-localmany-hpcc-localfile-pvc
+      prefix: "/var/lib/HPCCSystems/manydatacache2"
+      mode: [ sequential ]
+      numMounts: 3
```

Comment:
question: this could make sense, but we can't conceive of a scenario at the moment, where there is a 5 PVC plane, with a 3 mount alias can we?

it would still stripe (on read access) according to numDevices, but then need to manipulate the leading prefix according to numMounts... but which mount would have which stripe?